js/ui/autocomplete.js

```diff
@@ -101,6 +101,7 @@ const Autocomplete = DropDownList.inherit({
         this.callBase();
         this.$element().addClass(AUTOCOMPLETE_CLASS);
         this.setAria('autocomplete', 'inline');
+        this._dataSource.pageSize(this.option('maxItemCount'));
```

Comment:
Wouldn't it be better to set the initial pageSize here?
https://github.com/DevExpress/DevExtreme/blob/c0f577ef779a3f320cc9501c3e2bb154d2a600f5/js/ui/autocomplete.js#L164-L168

Reply:
thx, fixed it